### PR TITLE
PubMole v2

### PR DESCRIPTION
**Pull request by milainlambers**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+venv/
+__pycache__/
+Course8Informatica/__pycache__/
+__pycache__/*
```
Comment (milainlambers) [inline .gitignore:4]: nice

File: Course8Informatica/GenPanels_merged_DG-2.17.0.txt [hunk 2]
```diff
@@ -0,0 +1,4410 @@
+Symbol_HGNC	GenePanel
```
Comment (milainlambers) [inline Course8Informatica/GenPanels_merged_DG-2.17.0.txt:1]: Maybe rename the file in v3 :P

File: Course8Informatica/info_retriever.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        else:
+            text = text[:s+extra_index] + '<span class="span-results3" title="Not mentioned in either GenePanel or NCBI Human Gene Database"><b>'+ text[s+extra_index:e+extra_index] + '</b></span>' + text[e+extra_index:]
+
+            extra_index += 112
+
+    return text
+
+
+def return_genes(text):
+    """Rredict genes in a given text
```
Comment (milainlambers) [inline Course8Informatica/info_retriever.py:137]: patch spelling

File: Course8Informatica/pubmedsearchtool.py [hunk 4]
```diff
@@ -0,0 +1,113 @@
+from flask import Markup
+from Bio import Entrez, Medline
+Entrez.email = 'A.N.Other@example.com'
```
Comment (milainlambers) [inline Course8Informatica/pubmedsearchtool.py:3]: Could cause server to be blocked/banned without a way to fix it.

File: Course8Informatica/pubmedsearchtool.py [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+Entrez.api_key = "4bcbe73b19b2c6ca3c473e48a056f5dab709"
```
Comment (milainlambers) [inline Course8Informatica/pubmedsearchtool.py:4]: *coughs*

**Review by ghost — COMMENTED** [review]
mooi

**Review by ghost — APPROVED** [review]

File: config.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+from os import environ
+
```
Comment (milainlambers) [inline config.py:2]: Unused file

File: Course8Informatica/templates/testing/database_test.html [hunk 7]
```diff
@@ -0,0 +1,21 @@
+{% extends 'layout.html' %}
+
```
Comment (milainlambers) [inline Course8Informatica/templates/testing/database_test.html:2]: Unused file

File: wsgi.py [hunk 8]
```diff
@@ -0,0 +1,9 @@
+from Course8Informatica import app
+
+# Web Server Gateway Interface
+
+app = app()
+
+if __name__ == "__main__":
+
+    app.run(debug=True)
```
Comment (milainlambers) [inline wsgi.py:9]: Turn debug off

**Comment by morganatmo** [comment]
nice

**Pull request merged**